lib/helpers/helper_async.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Gyx.Python.HelperAsync do
+defmodule Gyx.Helpers.Python do
```

Comment:
the file name should reflect the name of the module.. it would be easier to search in the future, and not have to remember of which file contains what module..